.github/workflows/teleport-create-role.yaml

```diff
@@ -0,0 +1,54 @@
+name: Create teleport role
+on:
+  workflow_call:
+      inputs:
+        cluster_teleport:
+          description: "Address teleport cluster"
+          type: string 
+          default: "teleport.dev.aml-ai.net"
+          required: true
+        OTP_admin:
+          description: "Input OTP admin"
+          type: string
+          default: "123456"
```

Comment:
not sure that default is needed when required=true because we are forced to input something anyway

Reply:
this one i just input an example